### ISSUE TITLE
False positive on E1101: no-member when using "random.choices"

### ISSUE THREAD

**Issue by Pytonballoon810** [pylint-dev/pylint#9470]
### Bug description

```python
code snippet:

local_part = ''.join(random.choices(string.ascii_lowercase + string.digits, k=10))
```

### Command used

```shell
used as a VScode extension...
```

### Pylint output

```python
Module 'random' has no 'choices' member; maybe 'choice'?PylintE1101:no-member
```

### Expected behavior

random.choices is valid as seen here: 
```py
(variable) def choices(
    population: SupportsLenAndGetItem[_T@choices],
    weights: Sequence[float | Fraction] | None = None,
    *,
    cum_weights: Sequence[float | Fraction] | None = None,
    k: int = 1
) -> list[_T@choices]
```

### Pylint version

```shell
pylint 2023.10.1
python 3.12.0 64-bit
```

### OS / Environment

Windows 10 pro
